starter-code/javascript/chronometer.js

```diff
@@ -1,27 +1,39 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = 0;
+};
+
+
+Chronometer.prototype.startClick = function () {
+    var self = this;
+    window.setInterval(function() {
+        self.currentTime = self.currentTime + 1;
+    }, 1000); 
+}
+
+Chronometer.prototype.setMinutes = function () {
+  return Math.floor(this.currentTime / 60);
+};
+
+Chronometer.prototype.setSeconds = function () {
+  // var minutes = Math.floor(this.currentTime / 60);
+  return this.currentTime - this.setMinutes() * 60;
```

Comment:
Here you transform the minutes into seconds, but what you really want is to calculate the remaining seconds. One way is:
```
return self.currentTime % 60;
```